web/src/api/zfcp.ts

```diff
@@ -51,42 +51,42 @@ const probeZFCP = () => post("/api/storage/zfcp/probe");
 /**
  * Activates given controller
  * @param controllerId id of existing controller
- * @returns 
+ * @returns
  */
 const activateZFCPController = (controllerId: string) => post(`/api/storage/zfcp/controllers/${controllerId}/activate`);
 
 /**
  * Returns list of WWPNs for given controller
  * @param controllerId id of existing controller
- * @returns 
+ * @returns
  */
 const fetchWWPNs = (controllerId: string): Promise<string[]> => get(`/api/storage/zfcp/controllers/${controllerId}/wwpns`);
 
 /**
  * Returns list of LUNs for give controller and WWPN
- * @param controllerId 
```

Comment:
IMHO, having the name of the arguments without a description is not useful.

Reply:
Then it would be good to write something there :-)

Reply:
I think that with the defined types and parameter names it is obvious and do not need extra documentation but can add some if really needed.